openreview/venue_request/venue_request.py

```diff
@@ -20,7 +20,7 @@ def setup_venue_revision(self):
         revision_content['Additional Submission Options'] = {
             'order': 18,
             'value-dict': {},
-            'description': 'Configure additional options in the submission form. Valid JSON expected.'
+            'description': 'Configure additional options in the submission form. Use lowercase for the field names and underscores to represent spaces. The UI will auto-format the names, for example: supplementary_material -> Supplementary Material. Valid JSON expected.'
```

Comment:
Sounds good! can you add this description to all the other invitations that contain additional options?